### ISSUE TITLE
Carousel unselected cirlces do not have sufficient contrast (WCAG 1.4.11) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#200]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
Page:
Lesson 3: Observation Print

Description:
The carousel unselected slide controls has insufficient color contrast. The element was calculated to have the following traits
   • Foreground: `#E6E6E6`
   • Background: `#FFFFFF`
   • The contrast ratio is: 1.2:1

The minimum contrast ratio should be at least 3:1.
![image](https://user-images.githubusercontent.com/93537804/231589311-3dab882c-ddbd-4b60-b231-0701ea44e373.png)
*******************************************************************************
